### ISSUE TITLE
Closing the last editor should move focus back to explorer view

### ISSUE THREAD

**Issue by gauravahir28** [microsoft/vscode#108710]
Issue Type: <b>Bug</b>

Open a  file.
Close it using ctrl+F4.
The focus doesn't come again to the closed file.

VS Code version: Code 1.50.1 (d2e414d9e4239a252d1ab117bd7067f125afd80a, 2020-10-13T15:06:15.712Z)
OS version: Windows_NT x64 10.0.19041

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-8250U CPU @ 1.60GHz (8 x 1800)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>opengl: enabled_on<br>protected_video_decode: enabled<br>rasterization: enabled<br>skia_renderer: disabled_off_ok<br>video_decode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|7.91GB (2.92GB free)|
|Process Argv|--crash-reporter-id 3673b813-712f-4de4-aae0-a658c8663e0e|
|Screen Reader|yes|
|VM|0%|
</details><details><summary>Extensions (9)</summary>

Extension|Author (truncated)|Version
---|---|---
vscode-eslint|dba|2.1.10
java|red|0.68.0
sonarlint-vscode|Son|1.17.0
vscodeintellicode|Vis|1.2.10
vscode-java-debug|vsc|0.28.0
vscode-java-dependency|vsc|0.13.0
vscode-java-pack|vsc|0.10.0
vscode-java-test|vsc|0.24.2
vscode-maven|vsc|0.24.2

</details>

**Comment by bpasero** [comment]
I don't understand the issue here, when you close a file why should it focus it again?

**Comment by gauravahir28** [comment]
Hi @bpasero 
Thanks for commenting.
There might be other files as well in the file explorer tree.
Now, If I want to access them, I always have to close the file, navigate to the entire tree using keyboard, and then open it.
As I am a keyboard+screen reader user, If my focus gets to the closed file, then I can navigate around the siblings/childrens/parent files in the file explorer tree.

**Comment by gauravahir28** [comment]
E.G.
I have a tree/directory  structure like this:
Project(folder)
Project folder contains a folder "sub Project".
The "sub project" folder has 2 files.
"HelloWorld" and "HelloGithub".
If I Open "HelloWorld", and close it again, my focus should go to "HelloWorld".
The same doesn't happen now.
I all the way have to navigate this entire tree/directory to access "HelloGithub".

**Comment by isidorn** [comment]
@gauravahir28 thanks for providing feedback and sorry for the slow response I was on vacation.

So if you open both "HelloWorld" and "HelloGithub" then closing either one of them will focus the other one.
However if you open only one, we do not automatically open another file. That would be unexpeceted for most users.

If you wanted to open both, but did not maybe try changing this setting
`"workbench.editor.enablePreview": false`

**Comment by gauravahir28** [comment]
Hay @isidorn I guess I couldn't able to explain my issue.
Let me try again.

As you said in your comment, I don't want to open file explicitly.
My focus should be set to that file so that, I can press enter to open that file.
e.g.
If I open "Hello World", An if I close it again, my focus should be set to the title of that file in the file explorer tree  so that, I can navigate using down/up arrow in the file explorer tree and open the file using enter.
For now, If I close the file, my focus gets lost from the file explorer tree.

**Comment by isidorn** [comment]
I understand. So the gist you:
You want focus to move from editor area back to explorer when the last editor is closed.

This is a fair request, however we would not add it right now since it would break how users are used to focus in vscode. We rarely magically move focus around, since that would be unexpected for most users. Though I do understand your scenario.

**Comment by alexstine** [comment]
I really wish you would consider implementing this. I find it really annoying when I close the last editor and I am just stuck in an empty group. I have to move my focus back to the Explorer. I am also a blind screen reader user. VS Code has some powerful settings, why not add a setting for this?

Thanks.
